rules/busybox.bzl

```diff
@@ -118,6 +119,12 @@ def _make_resources_flag(
         ],
     )
 
+def _disable_warnings(ctx, args):
+    if (_flags.get(ctx).persistent_android_resource_processor or
```

Comment:
It turns out that we have some internal test rules that directly call into the busybox. It was a pain to manually add `_flags = [...]` to each of those rules, so I decided to add an early-fail here:

```
if not hasattr(ctx, "_flags"):
  return
```